config/openshift/rbac/vdo-controller-manager-metrics-service-rbac.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
```

Comment:
this is the service definition not the rbac. can you please check this

Reply:
Update